app/controllers/api/v1/student_projects_controller.rb

```diff
@@ -16,6 +16,19 @@ def update
     render_success(ProjectsSerializer.new(projects))
   end
 
+  def create
+    return render_error("Required parameter missing") if missing_params(required_create)
+    project = StudentProject.new(student_project_params) if student && instructor
+    if project.save
+      ProjectFeedback.create_feedback(project, params) if !params[:project_feedback].nil?
+      params[:mod] = project.project_template.mod
+      params[:id] = project.id
+      projects = Projects.new(params)
+      render_success(ProjectsSerializer.new(projects))
+    else
```

Comment:
Wondering if there's a render failure here?

Reply:
I will add that in for sad path next. Great q!